### ISSUE TITLE
What needs to be added to .vimrc to make this work?

### ISSUE THREAD

**Issue by seamusdemora** [bfrg/vim-cpp-modern#34]
I dunno'... maybe I'm just too dumb to use this plugin. I'm definitely a vim newbie, but I got nerdtree working with no sweat at all.

Anyway, I only want C syntax coloring, and I think I want it to load automatically. After installation (using git) I open one of my source files using `vim filename.c`. But all I get is some generic highlighting (which existed before I installed vim-cpp-modern). 

The README is no help, and after looking at some of the other issues I got no help from that. 

How do you start this... how do you get vim-cpp-modern to apply syntax coloring appropriate to C source code? Simple instructions, please (I'm retarded, OK).

**Comment by bfrg** [comment]
Did you enable syntax in `vimrc`?

```vim
filetype plugin indent on
syntax on
```

**Comment by bfrg** [comment]
1. After you opened `filename.c`, check the output of `:scriptnames`.
2. And can you post a screenshot?

**Comment by seamusdemora** [comment]
I've done as you suggested, but it doesn't look any different. Here's the screenshot:

<img width="1250" alt="Screenshot 2024-02-04 at 11 16 34 PM" src="https://github.com/bfrg/vim-cpp-modern/assets/14911811/a5b0e7c5-2f21-432c-a959-6fe20eb92ab6">

**Comment by bfrg** [comment]
Well, you show me a screenshot that contains almost only comments. `scriptnames` shows that the syntax file is loaded. What did you expect from the plugin? It looks like `main()` is highlighted, so it seems to work.

**Comment by seamusdemora** [comment]
@bfrg 

Apologies for the confusion - I thought it was the output of `scriptnames` that you were interested in. 

But no - at least I don't think it's working - vim-cpp-modern doesn't seem to be having any effect. The "syntax coloring" that you're seeing is from the "industry" coloring that I chose... using the color command IIRC. Here's another screenshot that might make that more clear: 
<img width="1250" alt="Screenshot 2024-02-05 at 9 12 13 PM" src="https://github.com/bfrg/vim-cpp-modern/assets/14911811/25de3d79-1ad0-4c48-8d74-781042370c9b">

**Comment by seamusdemora** [comment]
And here's a `tree` output showing where it's installed:

```
/home/pi/.vim
└── pack
    ├── colorschemes
    │   ├── opt
    │   └── start
    ├── others
    │   ├── opt
    │   └── start
    └── syntax
        ├── opt
        └── start
            └── vim-cpp-modern
                ├── after
                │   └── syntax
                │       ├── cpp.vim
                │       └── c.vim
                └── README.md

14 directories, 3 files
``` 

And here's my `~/.vimrc` file: 

```
set number
set cursorline
syntax on
set tabstop=4
set shiftwidth=4
set expandtab
colorscheme industry
set laststatus=2
set statusline=%F
filetype plugin indent on
syntax on
```

**Comment by bfrg** [comment]
The function names are highlighted, so the plugin seems to work.

If you add `let g:cpp_member_highlight = 1`, the plugin will also highlight struct members, as in `foo.bar` or `foo->bar`.

**Comment by seamusdemora** [comment]
> The function names are highlighted, so the plugin seems to work.
> 
> If you add `let g:cpp_member_highlight = 1`, the plugin will also highlight struct members, as in `foo.bar` or `foo->bar`.

Ah - so the syntax highlighting is *adjustable* then? 

The files I'm working with now don't have a `struct`, but I did notice that (in some of my comments in the code) that when I typed in the word `NOTE:`, that seems to be highlighted - at least a different color from the rest of my comment. 

Maybe I was expecting too much? I already had the "generic"(?) `syntax` option in `.vimrc` before adding `vim-cpp-modern`.
